### PR TITLE
HOLY MILK AND HOLY BILK

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Adds holy milk and holy bilk, as well as a crafting recipie for holy bilk using holy milk and beer.

## Why It's Good For The Game
Offers multiple options for "blessed liquids", allows oozelings (slime!!!!!!!!) to be purified without having to deal with water debuffs

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
add: holy milk and holy bilk
/:cl:

### CODE REVIEW

File: code/datums/components/cult_ritual_item.dm [hunk 1]
```diff
@@ -109,7 +109,7 @@
 
 	. = COMPONENT_CANCEL_ATTACK_CHAIN // No hurting other cultists.
 
-	if(!target.reagents.get_reagent(/datum/reagent/water/holywater))
+	if(!target.reagents.get_reagent(/datum/reagent/water/holywater) || !target.reagents.get_reagent(/datum/reagent/water/holywater/milk) || !target.reagents.get_reagent(/datum/reagent/water/holywater/bilk))
```
Comment (itsmeow) [inline code/datums/components/cult_ritual_item.dm]: does get_reagent not include subtypes?

**Review by itsmeow — CHANGES_REQUESTED** [review]
I don't like the way this is done, adding a bunch of ORs. It should be a var on regeant or something, or a signal, or a component, or an interface...

File: code/datums/components/cult_ritual_item.dm (hunk 1 above)
Comment (etherware-novice) [reply code/datums/components/cult_ritual_item.dm]: its just in case

**Review by etherware-novice — COMMENTED** [review]
wait i did review by accident

**Comment by bluezorua** [comment]
iirc oozelings dieing to holy water is an intentional feature and a part about their design

**Comment by Crossedfall** [comment]
Continuing to believe that 'bilk' is a bit out of place here. :key:

**Comment by etherware-novice** [comment]
then remove bilk

**Comment by ike709** [comment]
🔑 bloat

**Pull request closed without merging**